abce/trade.pyx

```diff
@@ -282,7 +282,7 @@ class Trade:
             offer.open_offer_status = 'polled'
             ret.append(offer)
         if sorted:
-            ret.sort(key=lambda objects: objects.price, reverse=descending, cmp=compare_with_ties)
+            ret.sort(key=lambda objects: objects.price, reverse=descending)
```

Comment:
Either:
1. delay porting to py3 / making the code compatible with py3 (default)
2. port to py3, at the expense of not cythonizing the sort function